network/network_interface.h

```diff
@@ -176,6 +176,48 @@ struct network_interface {
 	int32_t (*socket_recvfrom)(void *net, uint32_t sock_id,
 				   void *data, uint32_t size,
 				   struct socket_address *from);
+	/**
+	 * @brief Bind a specific port to a socket.
+	 *
```

Comment:
nitpick: extra line not needed

Reply:
The extra line is needed in order to separate the brief from the description in the doxygen documentation